### DISCUSSION TITLE
Search filter `status=` regressed on `master`

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#20287] [Q&A] [answered]
`2022.911.0` allows to type several letters of status to filter by it:
![изображение](https://user-images.githubusercontent.com/53872073/189908391-93afd749-8f85-4a0b-b199-38cb17c8d047.png)
![изображение](https://user-images.githubusercontent.com/53872073/189908503-324275d0-8367-4991-9046-ce840468884b.png)

Also, it allows you to type nothing. Pending maps will be shown.
![изображение](https://user-images.githubusercontent.com/53872073/189908062-0f83f1c2-ecde-4db6-8485-49eb4844d16f.png)

`566a61e`:
![изображение](https://user-images.githubusercontent.com/53872073/189909021-9ed7162a-0722-4e2c-9b2c-ede2c1e379a7.png)
![изображение](https://user-images.githubusercontent.com/53872073/189909058-57b5bbaa-64ee-45ed-a275-15a7c7f30f84.png)

Now you have to type the word fully, that's not good.

I suppose that #19275 may be related.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20289
